cmd/kubeadm/app/cmd/reset.go

```diff
@@ -81,13 +134,12 @@ func (r *Reset) Run(out io.Writer) error {
 	// Don't check for errors here, since umount will return a non-zero exit code if there is no directories to umount
 	exec.Command("sh", "-c", "cat /proc/mounts | awk '{print $2}' | grep '/var/lib/kubelet' | xargs umount").Run()
 
-	dirsToRemove := []string{"/var/lib/kubelet", "/var/lib/etcd", "/etc/kubernetes"}
-	fmt.Printf("Deleting the stateful directories: %v\n", dirsToRemove)
-	for _, dir := range dirsToRemove {
-		err := os.RemoveAll(dir)
-		if err != nil {
-			fmt.Printf("failed to remove directory: [%v]\n", err)
-		}
+	resetConfigDir("/etc/kubernetes/")
+
+	dirsToClean := []string{"/var/lib/kubelet", "/var/lib/etcd"}
+	fmt.Printf("Deleting contents of stateful directories: %v\n", dirsToClean)
+	for _, dir := range dirsToClean {
+		cleanDir(dir)
```

Comment:
I think that we might want to just do a `os.RemoveAll` here to minimize leftover